src/Ext/Aircraft/Body.cpp

```diff
@@ -50,3 +54,50 @@ bool AircraftExt::PlaceReinforcementAircraft(AircraftClass* pThis, CellStruct ed
 
 	return result;
 }
+
+DirType AircraftExt::GetLandingDir(AircraftClass* pThis, BuildingClass* pDock)
+{
+	auto const poseDir = static_cast<DirType>(RulesClass::Instance->PoseDir);
+
+	if (!pThis)
+		return poseDir;
+
```

Comment:
LGTM except for a minor improvement/bugfix:
In vanilla game when spawned aircrafts returns to the owner their facing are not aligned with the unit, it can be fixed right over here:
```suggestion

if (auto pOwner = pAir->SpawnOwner)
	return pOwner->PrimaryFacing.Current().GetDir();

```
if (auto pOwner = pAir->SpawnOwner)
			return pOwner->PrimaryFacing.Current().GetDir();